libipl/p10/common.C

```diff
@@ -287,3 +287,29 @@ int ipl_set_sbe_state_all_sec(enum sbe_state state)
 	}
 	return ret;
 }
+
+void ipl_plat_clock_error_handler(
```

Comment:
As commented previous commity this should be fisrt commit and enabe as part of previous commit.

Reply:
done